generate-cockroachdb-resources.md

```diff
@@ -224,10 +222,7 @@ To generate an HAProxy config file for a secure cluster, run the `cockroach gen
 ~~~ shell
 $ cockroach gen haproxy \
 --host=<address of any node in the cluster> \
---port=26257 \
---ca-cert=certs/ca.cert \
---cert=certs/root.cert \
---key=certs/root.key
```

Comment:
we need `--certs-dir` here as well (if we're being explicit about it in the docs). It does work if there are certs in the default directory